BenadrylAutomation/src/main/java/com/jnj/pageobjects/AllergiesPage.java

```diff
@@ -0,0 +1,127 @@
+/**
+ * @author Rashi Tiwari
+ * @date 21-March-24
+ */
+
+package com.jnj.pageobjects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+
+import com.jnj.actions.Action;
+import com.jnj.base.BaseClass;
+
+public class AllergiesPage extends BaseClass {
+	Actions actions;
+
+	/**
+	 * Constructor
+	 */
+	public AllergiesPage() {
+		PageFactory.initElements(driver, this);
+		actions = new Actions(driver);
+	}
+
+	/**
+	 * Locators
+	 */
+	@FindBy(xpath = "//div[@class='content-container node node--slide node--full node--slide--full']")
+	WebElement bannerImg;
+	
+	@FindBy(css = "h1#content-main")
+	WebElement bannerHeading;
```

Comment:
@RashiTiwari21 all locators should be private

Reply:
@goutamnaik007  all locators are private

Reply:
@vaibhavNagvekarSJI Verified